pom.xml

```diff
@@ -53,7 +59,8 @@
         <bundle.plugin.version>2.5.3</bundle.plugin.version>
         <cdi.tck-1-2.version>1.2.10.Final</cdi.tck-1-2.version>
         <cdi.tck-2-0.version>2.0.4.Final</cdi.tck-2-0.version>
-        <classfilewriter.version>1.1.2.Final</classfilewriter.version>
+        <!--1.2.2.Final is absolute minimum to eliminate IllegalAccess warning in JDK9+-->
+        <classfilewriter.version>1.2.2.Final</classfilewriter.version>
```

Comment:
If 1.2.2.Final is minimum, we can probably remove `org.jboss.weld.bean.proxy.ProxyFactory.CONFIGURABLE_ACCESS_FLAGS` and `org.jboss.weld.bean.proxy.ProxyFactory.CONFIGURABLE_CLASSLOADER_CONSTRUCTOR` too?

Anyway, this would mean we cannot create a patch for older WildFly releases, right?

Reply:
Hmm, I wanted to keep a note on why we needed that version specifically, but it's probably pointless.
As for the flags - probably yes, I can try and remove those, hope I won't break anything :)

As for patches - TBH I have no idea how this correlates with older WFLYs. I mean even WFLY must have had newer version for ages. We actually only compiled against older one.

Reply:
> We actually only compiled against older one.

Not really. We compiled against an older version but tried to leverage functionality from the newer ones.

Reply:
That's what I mean. WFLY had newer version for a long time. So I cannot see why we couldn't create a patch for, say, WFLY 12.